sda-commons-app-example/src/test/java/org/sdase/commons/spring/boot/web/app/example/OpenApiDocumentationTest.java

```diff
@@ -33,16 +39,61 @@ class OpenApiDocumentationTest {
 
   @Autowired private TestRestTemplate client;
 
+  // load only once, because servers are readded in subsequent requests
+  private static String openapi;
+
+  // verification of the current OpenAPI
+
   @Test
   void shouldHaveSameOpenApiInRepository() throws IOException {
     // receive the openapi.yaml from your service
-    String expected =
-        client.getForObject(
-            String.format("http://localhost:%s/api/openapi.yaml", port), String.class);
+    var expected = loadRawOpenApiAsYaml();
 
     // specify where you want your file to be stored
-    Path filePath = Paths.get("openapi.yaml").toAbsolutePath();
+    var filePath = Paths.get("openapi.yaml").toAbsolutePath();
 
     GoldenFileAssertions.assertThat(filePath).hasYamlContentAndUpdateGolden(expected);
   }
+
+  // integration testing of the OpenAPI customisation
+
+  @Test
+  void shouldNotContainServers() throws JsonProcessingException {
+    var openapi = loadRawOpenApiAsYaml();
```

Comment:
I would rename this var to not be confused with the class field `openapi`

Reply:
👍 fixed